pkg/api/team.go

```diff
@@ -77,6 +93,11 @@ func (a *API) setupTeamRoutes() {
 			return
 		}
 
+		// Remove owner from user slice for edit team form
```

Comment:
Kanskje bedre å si hvorfor man gjør det? For eks:
> Avoid duplicating owner as a user

Reply:
Enig, endrer